app/app.go

```diff
@@ -172,6 +189,10 @@ var (
 )
 
 func init() {
+	// Replace defaults
+	// manually update the power reduction by replacing micro (u) -> atto (a)
+	sdk.DefaultPowerReduction = cosmosevmtypes.MicroPowerReduction
+	stakingtypes.DefaultMinCommissionRate = math.LegacyZeroDec()
```

Comment:
This is [already zero](https://github.com/cosmos/cosmos-sdk/blob/e265bb9b42308dc70743b6200a70db9aafb70527/x/staking/types/params.go#L35)